README.md

```diff
@@ -2,8 +2,45 @@
 
 [![Build Status](https://travis-ci.org/xwp/go-tide.svg?branch=develop)](https://travis-ci.org/xwp/go-tide) [![Coverage Status](https://coveralls.io/repos/github/xwp/go-tide/badge.svg?branch=develop)](https://coveralls.io/github/xwp/go-tide?branch=develop) [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
+Tide services responsible for polling the WordPress.org APIs for themes and plugins to audit ("Sync Server"), for running audits and sending the results to the Tide API service ("Audit Server"), and for running Google Lighthouse audits against themes in the WordPress.org repository ("Lighthouse Server").
+
+## Table of Contents
+@todo
+
+### Setup
+@todo
+
+### Usage
+
+#### WPCS Auditing
```

Comment:
I think we should start to steer away from the term `audit` or `auditing`. Also, I would suggest these title changes:
```
### Usage
@todo

### PHPCS Server
@todo

#### WPCS
@todo

#### PHP Compatibility
@todo

### Lighthouse Server
This section outlines ...
```

Reply:
@valendesigns would you suggest renaming the `audit` package in `wptide/pkg`?  Perhaps `process` ?